### PR TITLE
add search input to mat-select

### PR DESCRIPTION
**Pull request by huadream**
for timezone
image

for account select
image

I am also doing the work for quota and usage support in quota-usage branch.
image
image

And a dashboard
image

### CODE REVIEW

**Comment by bwsw** [comment]
@huadream we check on monday. Thanks.

**Comment by huadream** [comment]
I should remind that I did not add the translation for ru.

**Comment by dron8552** [comment]
Add Russian translation:
```"SELECT_SEARCH": {
    "SEARCH": "Поиск...",
    "NO_MATCH": "Ничего не найдено"
}
```

**Comment by dron8552** [comment]
run test

**Comment by dron8552** [comment]
run tests

**Comment by dron8552** [comment]
run tests

File: src/app/security-group/sg-filter/sg-filter.component.ts [hunk 1]
```diff
@@ -44,4 +44,18 @@ export class SgFilterComponent {
   public get SecurityGroupViewMode() {
     return SecurityGroupViewMode;
   }
+
+  public ngOnChanges(changes: SimpleChanges) {
+    const accounts = changes['accounts'];
+    if (accounts) {
+      this.onAccountQueryChanged(this.accountQuery);
```
Comment (dron8552) [inline src/app/security-group/sg-filter/sg-filter.component.ts]: `ERROR in src/app/security-group/sg-filter/sg-filter.component.ts(51,39): error TS2339: Property 'accountQuery' does not exist on type 'SgFilterComponent'.`

File: src/app/security-group/sg-filter/sg-filter.component.ts [hunk 2]
```diff
@@ -44,4 +44,18 @@ export class SgFilterComponent {
...
+    }
+  }
+
+  public onAccountQueryChanged(accountQuery: string) {
+    const queryLower = accountQuery && accountQuery.toLowerCase();
+    this.accountsFiltered = this.accounts.filter(
```
Comment (dron8552) [inline src/app/security-group/sg-filter/sg-filter.component.ts]: `src/app/security-group/sg-filter/sg-filter.component.ts(57,10): error TS2339: Property 'accountsFiltered' does not exist on type 'SgFilterComponent'.`

**Comment by dron8552** [comment]
run tests

**Comment by ershovaelena** [comment]
To test (please see Requirements 1.411.30 doc): 
- REQ_VM_LIST_004
- REQ_STORAGE_LIST_003
- REQ_IMAGE_LIST_004
- REQ_SNAP_LIST_003
- REQ_SNAP_LIST_011
- REQ_FRL_LIST_003
- REQ_SSH_LIST_002
- REQ_ACT_003

**Pull request merged**